examples/demo-react-native/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -34,7 +55,11 @@ public ReactNativeHost getReactNativeHost() {
     @Override
     public void onCreate() {
         super.onCreate();
-        SoLoader.init(this, /* native exopackage */ false);
+        try {
+            SoLoader.init(this, OpenSourceMergedSoMapping.INSTANCE);
+        } catch (IOException e) {
```

Comment:
We can definitely rewrite this class in kotlin. Should make it easier to handle this case, among other things.